docs/sdks/sdk-ref-javascript.md

```diff
@@ -339,9 +359,7 @@ If you have already built an integration using a legacy callback function, you c
 
 ### getAdvertisingToken(): string
 
-Gets the current advertising token. 
-
-Before calling this function, be sure to call [init()](#initopts-object-void) and wait until your callback handler has received an `InitCompleted` event. 
+Gets the current advertising token. This function can be called without `init()` and will return the token if it is stored in local storage or a first-party cookie.  
```

Comment:
Remove future tense:

Gets the current advertising token. This function can be called without `init()`, and returns the token if it is stored in local storage or a first-party cookie.